components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -291,12 +291,31 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [button setValue:newValue forKey:keyPath];
           }
 
-        } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(title))]) {
-          [button setTitle:newValue forState:UIControlStateNormal];
+        } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(accessibilityHint))]) {
```

Comment:
Do you need to check for `respondsToSelector` as the code is doing for `setEnabled`?

Reply:
I don't believe so no.